### ISSUE TITLE
running

### ISSUE THREAD

**Issue by 111tobestronger** [yuzhiyang123/FL-BNN#1]
Hi, could you tell me why when training the mnist dataset, the train loss is first falling but soon rising and then continuously rising and no longer falling. My settings are initial learning rate=0.1 and numclients=100, which are exactly the same as in your article.

Namespace(alpha=1.25, batch_size=64, datano='0', dataset='MNIST', epochs=2500, evaluate=None, gpus=[2], input_size=None, lr=0.1, model='mnistnet', model_config='', momentum=0.9, numclients=100, optimizer='SGD', print_freq=10, results_dir='./results', resume='', save='fullfulliid', serveralg='Naive', start_epoch=0, type='torch.cuda.FloatTensor', weight_decay=0.0001, workers=8, workmode='fullfull')

![image](https://user-images.githubusercontent.com/90908254/154016569-2d878af3-d5c8-47dd-b182-b02975f3abe8.png)
![image](https://user-images.githubusercontent.com/90908254/154016634-fa3f4c91-adbe-431d-9422-2ea706a67965.png)
![image](https://user-images.githubusercontent.com/90908254/154016657-d2066e67-8fdd-4f16-9174-1a390916a7a4.png)

**Comment by VitaminyW** [comment]
I also encountered this problem, and it seems that I cannot reproduce the results of the paper.

**Comment by VitaminyW** [comment]
![image](https://github.com/yuzhiyang123/FL-BNN/assets/62011616/04b80619-25d1-4010-8258-b19ca5220d98)
